NEWS.md

```diff
@@ -1,5 +1,9 @@
 # mlr 2.14.0.9000
 
+## Breaking
+
+- Instead of a wide `data.frame` filter values are now returned in a long (tidy) `tibble`. This makes it easier to apply post-processing methods (like `group_by()`, etc) (@pat-s, #2456)
```

Comment:
I could imagine that this is actually something more people programmed against. So a breaking change here might irritate some people. Also I don't see the need to use a tibble here. If someone wants to do anything to the values one is free to transform and convert them as one pleases.
Regarding internal calculations: As we have data.table as a dependency - why don't we just use that?

Reply:
> I could imagine that this is actually something more people programmed against.

This refers to returning a long DF?
Changing this again would involve several hours since I have to re-arrange all outputs..

> Regarding internal calculations: As we have data.table as a dependency - why don't we just use that?

Besides one expection for which I failed with base R, we are not using any dplyr stuff internally.

> Also I don't see the need to use a tibble here. If someone wants to do anything to the values one is free to transform and convert them as one pleases.

Yes, sure. For now it is only used for printing, not internally. (i.e. the DF is coerced right before its returned). Which ofc makes no difference for the Import of tibble. I just hate it to print a DF that fills my console to Inf...

Reply:
> This refers to returning a long DF?

Yes

> Changing this again would involve several hours since I have to re-arrange all outputs..

Just saying that it could break someones code. It is something that could make a reverse dependency check worth it before going on cran.

> Besides one expection for which I failed with base R, we are not using any dplyr stuff internally.

Can you point me to it?

>  I just hate it to print a DF that fills my console to Inf...

Then just add the following to your `.Rprofile` No need to add a whole package to the dependencies.

```r
.First = function() {
  ee = new.env()
  if (interactive()) {
    if ("tibble" %in% rownames(installed.packages())) {
      ee$print.data.frame = function(x, ...) {
        tibble:::print.tbl(tibble::as_tibble(x), ...)
      }
    }
  }
  attach(ee, warn.conflicts = FALSE)
}
```

Reply:
> Just saying that it could break someones code. It is something that could make a reverse dependency check worth it before going on cran.

It will break code since the structure of the returned filter value is different, yes. 

> It is something that could make a reverse dependency check worth it before going on cran.

Yes, I always do that.

> Then just add the following to your .Rprofile No need to add a whole package to the dependencies.

Nice hack. I'll use it :) - and get rid of using _tibble_ then in the package.

Reply:
I updated it using melt from data.table

Reply:
:rocket: 	
so easy, damn...